vignettes/estimate_incubation.Rmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "Example to estimate incubation period"
+author: "Flavio Finger"
+date: "`r Sys.Date()`"
+output:
+   rmarkdown::html_vignette:
+     toc: true
+     toc_depth: 2
+vignette: >
+  %\VignetteIndexEntry{estimate_incubation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, echo = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.width=7,
+  fig.height=5,
+  fig.path="figs-overview/"
+)
+```
+
+
+Example using toy data.
+
+Load environment:
+
+```{r, echo = TRUE, message=FALSE}
+library(linelist)
+library(dplyr)
```

Comment:
For the vignette, you don't need to load all these packages, just linelist, epitrix, magrittr, and ggplot2